packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -171,6 +171,7 @@ class TextMapPropagator {
     carrier[traceparentKey] = spanContext.toTraceparent()
 
     ts.forVendor('dd', state => {
+      state.set('p', spanContext._spanId)
```

Comment:
```suggestion
     if (!spanContext._isRemote){
        # SpanContext was created by a ddtrace span. 
        # Last datadog span id should be set to the current span.
        state.set('p', spanContext._spanId)
     } else if (spanContext._trace.tags['_dd.parent_id']) {
        # Propagate the last Datadog span id set on the remote span.
        state.set('p', spanContext._trace.tags['_dd.parent_id'])
     }
```

Reply:
Wrong comment syntax. 😅 

```suggestion
      if (!spanContext._isRemote) {
        // SpanContext was created by a ddtrace span. 
        // Last datadog span id should be set to the current span.
        state.set('p', spanContext._spanId)
      } else if (spanContext._trace.tags['_dd.parent_id']) {
        // Propagate the last Datadog span id set on the remote span.
        state.set('p', spanContext._trace.tags['_dd.parent_id'])
      }
```